### ISSUE TITLE
Activation layer

### ISSUE THREAD

**Issue by arita37** [McGill-NLP/llm2vec#160]
Hi,

Thanks for the repo.

Just checking how to get the MLM layer activation logits 
from the bi-directionnal ?

Thanks
